src/app/zap-templates/zcl/data-model/chip/camera-av-settings-user-level-management-cluster.xml

```diff
@@ -0,0 +1,125 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/CameraAVSettingsUserLevelManagement.adoc
+Parameters: in-progress
+Git: 1.4-267-gc3d95b5c9
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="Cameras"/>
+  <struct name="MPTZPresetStruct" apiMaturity="provisional">
+    <cluster code="0x0552"/>
+    <item fieldId="0" name="PresetID" type="int8u"/>
+    <item fieldId="1" name="Name" type="char_string" length="32"/>
+    <item fieldId="2" name="Values" type="MPTZStruct"/>
+  </struct>
+
+  <struct name="MPTZStruct" apiMaturity="provisional">
+    <cluster code="0x0552"/>
+    <item fieldId="2" name="Pan" type="int16s" optional="true" default="0" min="-180" max="180"/>
+    <item fieldId="3" name="Tilt" type="int16s" optional="true" default="0" min="-180" max="180"/>
+    <item fieldId="4" name="Zoom" type="int8u" optional="true" default="1" min="1"/>
+  </struct>
+
+  <cluster apiMaturity="provisional">
+    <domain>Cameras</domain>
+    <name>Camera AV Settings User Level Management</name>
+    <code>0x0552</code>
+    <define>CAMERA_AV_SETTINGS_USER_LEVEL_MANAGEMENT_CLUSTER</define>
+    <description>This cluster provides an interface into controls associated with the operation of a device that provides pan, tilt, and zoom functions, either mechanically, or against a digital image.</description>
+    <client init="false" tick="false">true</client>
+    <features>
+      <feature bit="0" code="DPTZ" name="DigitalPTZ" summary="Digital PTZ support">
+        <optionalConform/>
+      </feature>
+      <feature bit="1" code="MPAN" name="MechanicalPan" summary="Mechanical Pan support">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="2" code="MTILT" name="MechanicalTilt" summary="Mechanical Tilt support">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="3" code="MZOOM" name="MechanicalZoom" summary="Mechanical Zoom support">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="4" code="MPRESETS" name="MechanicalPresets" summary="Mechanical saved presets support">
+        <optionalConform>
+          <orTerm>
+            <feature name="MPAN"/>
+            <feature name="MTILT"/>
+            <feature name="MZOOM"/>
+          </orTerm>
+        </optionalConform>
+      </feature>
+    </features>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="MPTZ" type="MPTZStruct" optional="true">MptzPosition</attribute>
```

Comment:
So why is it "MPTZ" in the name of the struct but "Mtpz" in the name of the attribute?  Should be consistent.  Please raise a spec issue.